schemas/witnet/witnet.proto

```diff
@@ -200,8 +200,7 @@ message TallyTransaction {
 }
 
 message MintTransaction {
-    fixed32 epoch = 1;
-    ValueTransferOutput output = 2;
+    ValueTransferOutput output = 1;
```

Comment:
So we have realized that the epoch field is actually useful, and shouldn't be removed. :sweat_smile: 

So just revert everything and add a comment to the schema, and we can mark the issue as fixed:

```suggestion
    // The epoch field is used to ensure that each MintTransaction has a unique hash
    fixed32 epoch = 1;
    ValueTransferOutput output = 2;
```

Reply:
done